app/components/image-cropper.js

```diff
@@ -0,0 +1,41 @@
+import Component from '@glimmer/component';
+import Cropper from 'cropperjs';
+import { action } from '@ember/object';
+
+export default class ImageCropperComponent extends Component {
+  currentImageCropCordinates = null;
```

Comment:
`currentImageCropCordinates` to `currentImageCropCoordinates`?

Reply:
Remove this, it was not required, thanks for noticing